src/extension/ios/plistBuddy.ts

```diff
@@ -54,13 +54,16 @@ export class PlistBuddy {
             } else {
                 productsFolder = path.join(iosProjectRoot, "build", "Build", "Products");
             }
-            const sdkType = simulator ? "iphonesimulator" : "iphoneos";
+            const sdkSuffix = simulator ? "simulator" : "os";
+            const deviceType = (scheme?.indexOf("-tvOS") ?? -1) > -1 ? "appletv" : "iphone";
```

Comment:
```suggestion
            const deviceType = 
                (scheme?.toLowerCase().indexOf("tvos") ?? -1) > -1 ? "appletv" : "iphone";
```

Reply:
I was under the impression that those suffixes were set automatically when creating the project, so the original version would be a tiny bit more robust (it works even if the project name contains the `tvos` string). I'm not an Xcode expert though, so I'll go with your suggestion. Just wanted to share the rationale behind the choice.